boilerplate/flyte/golang_test_targets/go-gen.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
why did this come in

Reply:
no idea. I simply ran `make generate` and this file was produced.

I get the same result if I run `make generate` in master.